docs/rules/key-spacing.md

```diff
@@ -162,6 +162,35 @@ var obj = {
 };
 ```
 
+#### Fine-grained control
+
+You can specify these options separately for single-line and multi-line configurations by organizing the options this way:
+
+```js
+"key-spacing": [2, {
+        "singleLine": {
```

Comment:
No, assuming you meant `singleLine` (and not `simpleLine`).

Since `singleLine` has overridden and specified `beforeColon` value as `false`, it will respect that value and won't force it for a single line cases.